src/components/Header/Header.jsx

```diff
@@ -22,7 +22,7 @@ import {
 } from 'reactstrap'
 import { UserRole } from '../../utils/enums'
 
-class Header extends React.Component {
+export class Header extends React.Component {
```

Comment:
I think we don't need to export the un-connected header component.

Reply:
That is used in the Header.test.jsx file to test some of the page structure.